### PR TITLE
[WIP] switch to MinVer

### PR DESCRIPTION
**Pull request by adamralph**
Another alpha test. Hope you don't mind.

This time I've added the project elements to required to replace the removed assembly attributes. As with last time, for the purpose of the alpha test, I've removed the building of the analyser meta-package.

FYI alpha 21 is the last planned alpha.

### CODE REVIEW

**Comment by adamralph** [comment]
All good! https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/builds/20563744#L135

**Comment by blairconrad** [comment]
Hey, @adamralph. We don't mind at all. It's nice getting a preview of what would change. I do like the shape of the change. It'd be _nice_ if there was a way to retain the metapackage, but maybe we should relookat dropping it.

**Comment by adamralph** [comment]
@blairconrad there could be a way to keep it. After I release the first beta (probably tomorrow) I'll send another PR and see if I can work out a way to do it.

**Comment by blairconrad** [comment]
Oh, very cool. If nothing else, the minver-cli should allow this functionality, no?

**Comment by adamralph** [comment]
I guess it could, but I'm not really sure about mixing `MinVer` and `minver-cli` in the same project. The inputs to each would have to kept in sync somehow, and that might not be so easy.

File: src/CommonAssemblyInfo.cs [hunk 1]
```diff
@@ -1,10 +1,7 @@
-using System;
 using System.Reflection;
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyCompany("Patrik Hägne")]
-[assembly: AssemblyCopyright("Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en-US")]
```
Comment (thomaslevesque) [inline src/CommonAssemblyInfo.cs]: While we're at it, we should probably consider moving everything from AssemblyInfo.cs to csproj or Directory.Build.props (except `ComVisibleAttribute`, which is required by [CA1017](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1017-mark-assemblies-with-comvisibleattribute?view=vs-2017) and can't be set in csproj as far as I know)

Comment (adamralph) [reply src/CommonAssemblyInfo.cs]: I think that's a good move. Probably best done now, regardless of MinVer.

**Pull request closed without merging**